data/taurus_test_users.csv

```diff
@@ -0,0 +1,15 @@
+email,password,anonym_id
+student1@raccoongang.com,p1,d3d9446802a44259755d38e6d163e820
```

Comment:
There is no warring about actually this is just login info anyone could register in our system with there own credentials and have the same rights.